app/views/application/_nav_right.html.slim

```diff
@@ -22,7 +22,7 @@ ul.nav.navbar-nav.navbar-right
       li class=active_if(community_path) = link_to 'Community', community_path
   li class=active_if(page_path('help')) = link_to 'Help', page_path('help')
 
-  - if signed_in?
+  - if signed_in_and_confirmed?
```

Comment:
I was casually testing the branch with an unconfirmed user and noticed this condition was causing the Sign in button to read "Sign in" and not show me the menu even if I was already signed in. Maybe this check should really be `signed_in?` and return true even if the user is unconfirmed.

Reply:
Good catch, thank you!